functions/views.py

```
@@ -1,3 +1,128 @@
+                and {user.id: user for user in AppUser.objects.filter(id__in=user_ids)}
+                or {}
+            )
+
+            for account, scope, scope_parts in account_scopes:
+                toolkit_name = get_toolkit_name_by_slug(account["toolkit"]["slug"])
+                published_run_id = scope_parts.get(ScopeParts.saved_workflow)
+                published_run = (
+                    published_run_id and published_runs.get(published_run_id) or None
+                )
+                user_id = scope_parts.get(ScopeParts.member)
+                user = user_id and users.get(int(user_id)) or None
```

Ruff (0.14.4)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)